wiki/Gameplay/Input_device/de.md

```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+---
+
+# Eingabegerät
+
+Ein **Eingabegerät** ist ein Gerät, das zur Steuerung der Eingabe auf einem Computer oder in einer bestimmten Software verwendet wird. Im osu!-Wiki bezieht sich dieser Begriff speziell auf Geräte, die in osu! verwendet werden können.
```

Comment:
`osu!-Wiki`: Every time I'm reading terms such as this, I'm unsure which version to choose:
- `osu!-Wiki`
- `osu! wiki`
- `osu! Wiki`

Reply:
pretty sure i've used `osu!-Wiki` in multiple articles now, so gonna leave it like that. i dont think it matters that much anyway. all three are fine

Reply:
Personally I use `osu! wiki` on articles on the French side, since that's how it's written.